### PR TITLE
ready to support http client timeout feature

### PR DESCRIPTION
**Pull request by mrdivyansh**
I am supposing that @bbansal is little bit busy. So, I have done some changes as per @vincentwoo 's suggestions on @bbansal pull request. We need HTTP client support as soon as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ gem install firebase
 ```ruby
 base_uri = 'https://<your-firebase>.firebaseio.com/'
 
-firebase = Firebase::Client.new(base_uri)
+firebase = Firebase::Client.new(base_uri, { :connect_timeout => , :send_timeout})
```
Comment (vincentwoo) [inline README.md]: This line isn't well-formed Ruby

File: README.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ firebase = Firebase::Client.new(base_uri, secret_key)
 response = firebase.push("todos", { :name => 'Pick the milk', :priority => 1 })
 ```
 
+If you have to configure HTTP Client timeout, set times as follows:
```
Comment (vincentwoo) [inline README.md]: This kind of reads like ESL, I would explain what each of those options do, or link to the appropriate page in httpclient's documentation.

**Comment by vincentwoo** [comment]
Made a couple doc-related comments, seems good to go once you fix those.

**Comment by mrdivyansh** [comment]
I have added description. I am supposing now it's ready to merge.

**Comment by vincentwoo** [comment]
These docs are really bad and the tests don't even pass (did anyone even run them?). I'm closing this for now, but I'll try to make my own commit for passing args to httpclient.

**Comment by mrdivyansh** [comment]
I haven't run test cases. I was tried to full-fill your requirement over
@bbanasl pull request.

Sure do it your own and publish soon.
Thanks

**Pull request closed without merging**